gbalink/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
 include_directories(. ./../Runtime ${ATHENA_INCLUDE_DIR} ${JBUS_INCLUDE_DIR})
-add_executable(gbalink main.cpp)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -ldl")
```

Comment:
It works! However hecl/include/hecl/SystemChar.hpp triggers new errors now:
'#include <string.h>' and '#include <algorithm>' should be added for 'strlen' and 'std::transform'. Should I make a PR in hecl ?